libs/common/blob.hpp

```diff
@@ -84,25 +85,29 @@ namespace iroha {
       return res;
     }
 
-    static blob_t<size_> from_string(const std::string &data) {
+    static blob_t<size_> from_raw(const uint8_t data[size_]) {
```

Comment:
```suggestion
    static blob_t<size_> from_raw(const byte_t data[size_]) {
```

Reply:
I would leavi it as is because
https://github.com/hyperledger/iroha/blob/e009c9395485fcb3d74b4da7ef7990cc5d052456/shared_model/cryptography/blob.hpp#L28

Reply:
`cryptography/blob.hpp` is not related to `common/blob.hpp`

Reply:
now I see, thanks